src/ontology/mondo-edit.obo

```diff
@@ -258261,6 +258283,7 @@ property_value: confidence "0.5294117647058822" xsd:double
 [Term]
 id: MONDO:0013159
 name: muscular dystrophy-dystroglycanopathy (congenital with intellectual disability), type B1
+synonym: "congenital muscular dystrophy-POMT1 related" EXACT CLINGEN_PREFERRED []
```

Comment:
could you please add a dbxref here:
https://clinicalgenome.org/affiliation/50061/